neuronsimulator/templates/home.html

```diff
@@ -4,12 +4,20 @@
 <div>
     <h3>Neuron Simulator Dashboard</h3>
     <br>
-    <h5>Note: placeholder; will add contents later</h5>
+    <h4>Overview</h4>
+    <p>Note: need feedback from the lab about this section </p>
```

Comment:
NB. I try and avoid placeholders like this, but if you prefer to use them, I suggest employing one or more these tricks I have used in the past to easily find them in the future:
- Comment out the code - or put separate comments above and below.
- Use a keyword that you can use to find placeholders later, e.g. `#PLACEHOLDER#` or `#TODO#`.
- Create a github issue that either/both links to the line of code or add a link to the issue in the comment in the code.